Assets/Scripts/Flight Model/SpaceshipController.cs

```diff
@@ -194,7 +194,7 @@ IEnumerator AngularDragModifier()
         }
     }
 
-    void Update()
+    void FixedUpdate()
```

Comment:
I think with this change the Time.deltaTime in these two helpers needs to change to Time.fixedDeltaTime, since its now running on a different update frequency that may be different from the actual frame rate regular delta time reports.